packages/mobile/src/web3/saga.ts

```diff
@@ -77,13 +79,17 @@ export function* checkWeb3SyncProgress() {
         if (latestBlock && latestBlock.number > 0) {
           yield put(completeWeb3Sync(latestBlock.number))
           Logger.debug(TAG, 'checkWeb3SyncProgress', 'Sync is complete')
+          millisecs = Date.now() - initTime
+          CeloAnalytics.track(CustomEventNames.sync_complete, { millisecs })
```

Comment:
This method gets called potentially many times throughout the lifetime of the app. We probably don't want to spam out events for each micro-sync right? We should find a different strategy for tracking sync time

Reply:
We can do a Moving Average every N micro-sync and then post every M micro-sync. What do you think.

Reply:
I did it now with only one parameter:  N = M.

Reply:
For example track() is called on start sync and once every 100 syncs. Mean and spot time elapsed since sync start is reported.